drivers/service_scanning/app/models/filters/scan.rb

```diff
@@ -0,0 +1,51 @@
+###
+# Copyright 2016 - 2020 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Filters
```

Comment:
Should this be in `ServiceScanning::Filters`?

Reply:
See the bottom of the file.  Again, having issues with the dev environment figuring out where things are supposed to be coming from and what what the namespaces are.